### PR TITLE
[support] Update polkadot dependencies

### PR DESCRIPTION
**Pull request by lvndry**
### 📝 Description

Update polkadot related dependencies in LLC, LLD and LLM

### ❓ Context

- **Impacted projects**: `LLC` `LLM` `LLD` 
- **Linked resource(s)**: `https://ledgerhq.atlassian.net/browse/LIVE-8984`

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-mobile/ios/ledgerlivemobile.xcodeproj/project.pbxproj
Comment (haammar-ledger) [inline apps/ledger-live-mobile/ios/ledgerlivemobile.xcodeproj/project.pbxproj:1]: Is this required?

File: libs/coin-polkadot/package.json [hunk 1]
```diff
@@ -53,22 +53,22 @@
...
-    "@types/invariant": "^2.2.2",
-    "@types/jest": "^29.2.4",
-    "@types/lodash": "^4.14.191",
-    "jest": "^28.1.1",
-    "ts-jest": "^28.0.5"
+    "@types/invariant": "^2.2.35",
+    "@types/jest": "^29.5.5",
+    "@types/lodash": "^4.14.198",
+    "jest": "^28.1.3",
+    "ts-jest": "^28.0.8"
```
Comment (haammar-ledger) [inline libs/coin-polkadot/package.json:71]: Out of scope of this PR, right?

File: libs/coin-polkadot/package.json [hunk 2]
```diff
@@ -53,22 +53,22 @@
...
-    "@polkadot/util-crypto": "10.3.1",
+    "@polkadot/types": "^10.9.1",
+    "@polkadot/types-known": "^10.9.1",
+    "@polkadot/util": "^12.5.1",
+    "@polkadot/util-crypto": "12.5.1",
     "bignumber.js": "^9.1.2",
-    "expect": "^27.4.6",
-    "invariant": "^2.2.2",
+    "expect": "^27.5.1",
+    "invariant": "^2.2.4",
```
Comment (haammar-ledger) [inline libs/coin-polkadot/package.json:62]: Same

**Review by hedi-edelbloute — COMMENTED** [review]
please update only what is necessary

**Comment by haammar-ledger** [comment]
@lvndry please pay attention to the PR's title & description
- polkdatot :eyes: 
- provide the JIRA ticket link
Thanks

**Comment by haammar-ledger** [comment]
@lvndry about bot runs: while a little funds remain on `Nitrogen`, the seed we officially use to test Polkadot is actually `Oxygen` (Nitrogen runs daily, for most coins that are cheap enough, Oxygen runs weekly, for all chains that are too expensive, including DOT).

Please run the Oxygen bot a few multiple times, the ideal is to make sure that all types of mutations are run at least once, which you can check in the report comment in the sections "_Details of the X mutations_" and "_Details of the Y uncovered mutations_":
image

However in the case of Polkadot, staking flows are actually not really used and hard to run with the bot... so please just make sure that at least **send ~50%** and **send max** have been run. Thanks!

**Review by haammar-ledger — APPROVED** [review]

**Review by hedi-edelbloute — APPROVED** [review]
Lgtm, please test operations not covered by the bot :
```
send max: send is too low to activate account (1), balance is too low (1)
bond - bondExtra: not enough balance to bond (1), balance is too low (1)
unbond: can't unbond (1), balance is too low (1)
rebond: can't rebond (1), balance is too low (1)
nominate: can't nominate (1), balance is too low (1)
withdraw: nothing to withdraw (1), balance is too low (1)
```

**Comment by lvndry** [comment]
Couldn't test bound/unbound because at least 250 DOT minimum are needed. Otherwise everything works fine

**Pull request closed without merging**